partners.yml

```diff
@@ -457,6 +457,10 @@ components:
           type: boolean
           description: Indicate if customer accepted can be profiled by Moank to take a credit decision
           example: true
+        bankAccountId:
+          type: string
+          description: Tink code fetched with the Tink link.
+          example: '123456789'
```

Comment:
The bank account id is suppoed to be a mongoId of one of the bank accounts of the user 🤔 Why did you think it was that ? 😅